src/parser/core/modules/Tincture.ts

```diff
@@ -0,0 +1,12 @@
+import {t} from '@lingui/macro'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {BuffWindowModule} from 'parser/core/modules/BuffWindow'
+
+export class Tincture extends BuffWindowModule {
+	static handle = 'tincture'
+	static title = t('core.tincture.title')`Tinctures`
+
+	buffAction = ACTIONS.INFUSION_INT
```

Comment:
Why not STR? :mnknife:

Reply:
If you want STR, do it yourself :P  Also, that part never shows up anywhere anyway and is just needed because buffAction is abstract.

Reply:
Aight, if you can rebase I'll re-approve and merge then.